hibernate-core/src/main/java/org/hibernate/bytecode/spi/BytecodeEnhancementMetadata.java

```diff
@@ -59,4 +59,6 @@ LazyAttributeLoadingInterceptor injectInterceptor(
 	LazyAttributeLoadingInterceptor extractInterceptor(Object entity) throws NotInstrumentedException;
 
 	boolean hasUnFetchedAttributes(Object entity);
```

Comment:
The addition of `isAttributeLoaded` will break applications that provide a custom `EntityPersister` unless the implementation extends `AbstractEntityPersister`.